apps/status-api/src/controllers/OracleStatusController.ts

```diff
@@ -31,12 +31,12 @@ export class OracleStatusController {
    * To provide the status of each ticker determined by the number of oracles responded given the ticker id
    *
    * @return {Promise<OracleStatus>}
-   * @param token
-   * @param currency
+   * @param pair
    */
-  @Get('/ticker/:token-:currency')
-  async getOracleRespondStatus (@Param('token') token: string, @Param('currency') currency: string): Promise<{ status: OracleStatus }> {
-    return await this.cachedGet(`oracle-${token}-${currency}`, async () => {
+  @Get('/ticker/:pair')
+  async getOracleRespondStatus (@Param('pair') pair: string): Promise<{ status: OracleStatus }> {
+    const [token, currency] = pair.split('-')
```

Comment:
Should probably regex validate the pair to fit certain constraints. Since now you're allowed to `A-B-C` and when `.split('-')` you get 3 items in the array.

Reply:
I have added the regex validation to only allow 2 items; 1 dash before it gets split to token & currency.